src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -636,7 +633,7 @@ public String getInvalidityInfoForTimeForVisibilityStartAndSessionStart(
      * @return Error string if {@code visibilityStart} is before {@code resultsPublish}
      *         Empty string if {@code visibilityStart} is after {@code resultsPublish}
      */
-    public String getInvalidityInfoForTimeForVisibilityStartAndResultsPublish(
+    public static String getInvalidityInfoForTimeForVisibilityStartAndResultsPublish(
```

Comment:
You need to update all the private methods too. See [this](https://stackoverflow.com/questions/2042813/calling-non-static-method-in-static-method-in-java) to understand why.